components/bio-formats-tools/src/loci/formats/tools/ImageInfo.java

```diff
@@ -204,7 +204,10 @@ public boolean parseArgs(String[] args) {
         else if (args[i].equals("-separate")) separate = true;
         else if (args[i].equals("-expand")) expand = true;
         else if (args[i].equals("-cache")) cache = true;
-        else if (args[i].equals("-omexml")) omexml = true;
+        else if (args[i].equals("-omexml")) {
+          omexml = true; 
+          validate = true;
```

Comment:
The main issue with setting both parameters in this block is that it makes the order of the arguments matter, i.e. the two following commands do not have the same functionality:

```
./showinf  -nopix -novalid -omexml file.ome.tiff
./showinf  -nopix -omexml -novalid file.ome.tiff 
```

As `validate` is conditional to `omexml`, thoughts about setting the default value to `true` and leaving `-validate/-novalid` to override it as in `ome-files info`?